exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -397,17 +412,42 @@ private static Exemplar toProtoExemplar(
         Exemplar.newBuilder()
             .setValue(exemplar.getValue())
             .setTimestamp(convertTimestamp(exemplar.getTimestamp()));
+    @javax.annotation.Nullable String traceId = null;
+    @javax.annotation.Nullable String spanId = null;
     for (Map.Entry<String, String> attachment : exemplar.getAttachments().entrySet()) {
-      // TODO: add support for special attachments once Stackdriver Monitoring accepts them.
+      String key = attachment.getKey();
+      String value = attachment.getValue();
+      if (ExemplarUtils.ATTACHMENT_KEY_TRACE_ID.equals(key)) {
+        traceId = value;
+      } else if (ExemplarUtils.ATTACHMENT_KEY_SPAN_ID.equals(key)) {
+        spanId = value;
+      } else { // Everything else will be treated as plain strings.
+        builder.addAttachments(
```

Comment:
Should we use "DroppedLabels"? not sure what is that about.

Reply:
> not sure what is that about.

Me either, but I think that's a special attachment and probably not everything fits into it. I left a [TODO](https://github.com/census-instrumentation/opencensus-java/pull/1771/files#diff-d2886f5b11a5e3be7a00a8f5fff5a1f0R126) to support it once I have a better understanding on it.